app/views/collection_numbers/_form.html.erb

```diff
@@ -1,9 +1,12 @@
-<% # form_tag(add_query_param(action: :new, id: @observation.id)) do %>
-<%= form_with model: :collection_number do |f| %>
+<% id = @observation&.id
+   id_path_segment = id ? "/#{id}" : nil
+   url = "#{collection_numbers_path}#{id_path_segment}" +
```

Comment:
Tests expect the form to be submitted to a url that includes `back` and `q` parameters.
Is there a more idiomatic way to make the form do that?

Reply:
https://mushroomobserver.org/collection_number/show_collection_number takes me to the collection number index page so it at least seem plausible.  As long as you're tweaking this, I'd rather see this use the correct path methods than hacking together an explicit URL.

Reply:
Yes - `collection_numbers_path(id)` should produce what you want.

Reply:
I don't need to tweak this. The new form syntax handles it fine.